### PR TITLE
Python CI.

### PR DESCRIPTION
**Pull request by lukesneeringer**
This commit adds a CI setup for Python. WIP.

_In its current state, the actual tests should fail_, because the existing code on master right now has failing tests.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -3,9 +3,86 @@ version: 2
 jobs:
   build:
     docker:
-      - image: ubuntu:16.04
+      - image: googleapis/curl:latest
```
Comment (geigerj) [inline .circleci/config.yml]: What/where is this image?

File: .circleci/config.yml [hunk 2]
```diff
@@ -3,9 +3,86 @@ version: 2
...
     steps:
       - run:
-          name: No-op.
-          command: echo "This command did nothing. It exists to configure CircleCI so that PRs that supply actual configuration will work."
-    working_directory: /var/code/googleapis
+          name: Add command to be able to trigger jobs.
+          command: |
+            touch trigger_job.sh && chmod a+x trigger_job.sh
```
Comment (geigerj) [inline .circleci/config.yml]: Why are you creating/storing this as a file instead of just defining a function?

**Comment by lukesneeringer** [comment]
Quick brain dump of our plan here:

#### Status Quo

Right now, all but three APIs do not have tests, but three (pubsub, speech, language) were committed while work on the tests was in progress, and they fail.
  
#### The Plan

After discussing with @geigerj, what we intend to do is:

  * Merge this. CI will intentionally go red.
  * Work on #233, #234, and #235, in order. For each we will show and document that the appropriate tests went from red to green. Whichever one we do last should be a green build.
  * Bump all the remaining GAPICs, with proper CI verification for each.

File: .circleci/config.yml (hunk 2 above)
Comment (lukesneeringer) [inline .circleci/config.yml]: Every command is independent; the function definition would not survive to the next step.

File: .circleci/config.yml (hunk 1 above)
Comment (lukesneeringer) [inline .circleci/config.yml]: On Docker Hub. It is two lines.

Comment (lukesneeringer) [inline .circleci/config.yml]: If you are curious, the entire Dockerfile is:

```Dockerfile
FROM alpine:latest
RUN apk add --update curl \
  && rm -rf /var/cache/apk/*
```

**Pull request merged**